analysis_driver/process_trigger.py

```diff
@@ -51,11 +51,11 @@ def _transfer_to_int_dir(dataset, from_dir, to_dir, repeat_delay):
     rsync_cmd = 'rsync -aqu --size-only --partial %s %s' % (os.path.join(from_dir, dataset), to_dir)
 
     while dataset_status(dataset) != 'transferring, rta complete':
-        exit_status += executor.execute([rsync_cmd], env='local').join()
+        exit_status += executor.execute([rsync_cmd], job_name='rsync', run_id=dataset).join()
         sleep(repeat_delay)
 
     # one more rsync after the RTAComplete is created. After this, everything should be synced
     sleep(repeat_delay)
-    exit_status += executor.execute([rsync_cmd], env='local').join()
+    exit_status += executor.execute([rsync_cmd], job_name='rsync', run_id=dataset).join()
```

Comment:
The default executor walltime is 8 hrs - should we specify something a bit higher here?